packages/format-currency/src/index.ts

```diff
@@ -6,17 +6,20 @@ export { getCurrencyDefaults };
 
 export { CURRENCIES } from './currencies';
 
+export * from './types';
+
 /**
  * Formats money with a given currency code
  *
- * @param   {number}     number              number to format
+ * @param   {number}     number              number to format; assumed to be a float unless isSmallestUnit is set.
  * @param   {string}     code                currency code e.g. 'USD'
  * @param   {object}     options             options object
  * @param   {string}     options.decimal     decimal symbol e.g. ','
  * @param   {string}     options.grouping    thousands separator
  * @param   {number}     options.precision   decimal digits
  * @param   {string}     options.symbol      currency symbol e.g. 'A$'
  * @param   {boolean}    options.stripZeros  whether to remove trailing zero cents
+ * @param   {boolean}    options.isSmallestUnit  true if the number is an integer in the currency's smallest unit.
```

Comment:
Nit: Prior to this PR, the docblock sections were aligned. Should we maintain that?

```suggestion
 * @param   {number}     number                  number to format; assumed to be a float unless isSmallestUnit is set.
 * @param   {string}     code                    currency code e.g. 'USD'
 * @param   {object}     options                 options object
 * @param   {string}     options.decimal         decimal symbol e.g. ','
 * @param   {string}     options.grouping        thousands separator
 * @param   {number}     options.precision       decimal digits
 * @param   {string}     options.symbol          currency symbol e.g. 'A$'
 * @param   {boolean}    options.stripZeros      whether to remove trailing zero cents
 * @param   {boolean}    options.isSmallestUnit  true if the number is an integer in the currency's smallest unit.
```